authorization/token/token.go

```diff
@@ -28,6 +29,7 @@ const (
 
 	// Token Statuses
 
+	TOKEN_STATUS_VALID         = 0
```

Comment:
I'd prefer not to define such a constant - a valid token is a token that has none of the other status flags, we have a function which abstracts this test in the `Token` repository object - `Valid()`, which is a preferable way to test token validity.

Reply:
ok now that I know I can use `tok.Token().Valid()` as you showed above. Although, the status flag being `0` by default, I don't see a problem with having a constant to reflect this value. Anyways, I'll remove it for now.